README.md

```diff
@@ -1,2 +1,5 @@
 # hello-world
-this is my first folder on Github. よろしくお願いします。
+This is my first folder on Github. よろしくお願いします。
+I studied electrical engineering in college and am currently studying programming and Japanese in my free time. 
+My goal is to obtain a programming job in Tokyo, where I am living now. 
```

Comment:
I moved back to the states and now wish to obtain a remote work engineering job!